### PR TITLE
docs(configuration): webpack-dev-server v4

### PR DESCRIPTION
**Pull request by snitin315**
Migrate Docs for v4.

### CODE REVIEW

**Comment by snitin315** [comment]
I think all major breaking changes are done, and `rc` is released for webpack-dev-server, So I will update the docs here. 

/cc @alexander-akait

**Comment by alexander-akait** [comment]
Yep, maybe we found something here what are missed in our migration guide, feel free to work on this PR, thanks for help

File: src/content/configuration/dev-server.md [hunk 1]
```diff
@@ -458,94 +460,130 @@ module.exports = {
...
+npx webpack serve --https
+```
+
+To pass your own certificate via the CLI use the following options
+
+```bash
+npx webpack serve --https-key ./path/to/server.key --https--cert ./path/to/server.crt --https-cacert ./path/to/ca.pem
+```
+
+## `devServer.headers`
```
Comment (alexander-akait) [inline src/content/configuration/dev-server.md:586]: Can be `Function`

File: src/content/configuration/dev-server.md [hunk 2]
```diff
@@ -615,17 +653,17 @@ module.exports = {
...
+Usage via the CLI:
 
 ```bash
 npx webpack serve --history-api-fallback
 ```
 
 For more options and information, see the [connect-history-api-fallback](https://github.com/bripkens/connect-history-api-fallback) documentation.
 
-## `devServer.host`
+## devServer.host
```
Comment (alexander-akait) [inline src/content/configuration/dev-server.md:664]: We have `local-ip`/`local-ipv4`/`local-ipv6`

File: src/content/configuration/dev-server.md [hunk 3]
```diff
@@ -615,17 +653,17 @@ module.exports = {
...
 
-`string = 'localhost'`
+`string`
```
Comment (alexander-akait) [inline src/content/configuration/dev-server.md:666]: We have `local-ip`/`local-ipv4`/`local-ipv6`, make sense to mark, it I will fix schema in dev server

Comment (snitin315) [reply src/content/configuration/dev-server.md:666]: I just noticed it too, need to update the schema. I can send a PR.

Comment (alexander-akait) [reply src/content/configuration/dev-server.md:666]: Already WIP on this :+1: Also will check other values

**Comment by snitin315** [comment]
Reday for review - 

https://webpack-js-org-git-fork-snitin315-dev-server-new-webpack-docs.vercel.app/configuration/dev-server/

**Comment by snitin315** [comment]
I will rebase

**Comment by snitin315** [comment]
I will merge it now, we will improve it more with future PRs.

**Pull request merged**